cps_data/finalprep.py

```diff
@@ -381,8 +384,11 @@ def benefits(data, other_ben):
     data['ssi_ben'] = data['ssi_ben'].astype(np.int32)
     data['snap_ben'] = data['snap_ben'].astype(np.int32)
     data['vet_ben'] = data['vet_ben'].astype(np.int32)
+    data['tanf_ben'] = data['tanf_ben'].astype(np.int32)
+    data['wic_ben'] = data['wic_ben'].astype(np.int32)
+    data['housing_ben'] = data['housing_ben'].astype(np.int32)
     data['e02400'] = data['e02400'].astype(np.int32)
-    data['e02300'] = data['e02300'].astype(np.int32)
+    data['e02300'] = data['ui_ben'].astype(np.int32)
```

Comment:
@andersonfrailey So are we replacing e02300 with the imputed unemployment compensation? In other words, we're using imputed imputed UI for tax calculations as well? I thought it was the other way around but total fine with this if the tax liabilities look ok. @MattHJensen Do you think it's fine  to use imputed benefit to replace e02300? Anderson previously checked the total of imputed UI is 93% of e02300.